src/components/submission/feed-entry.vue

```diff
@@ -135,13 +146,29 @@ export default {
   methods: {
     updateOrEditIcon(state) {
       return `${this.reviewStateIcon(state)} submission-feed-entry-icon`;
+    },
+    entityLabel(entry) {
+      if ('entity' in entry.details)
+        return entry.details.entity.label;
+      return '';
+    },
+    entityDataset(entry) {
+      if ('entity' in entry.details)
+        return entry.details.entity.dataset;
```

Comment:
This is also available on the actee, right? `entry.actee.name`?

Reply:
It's not actually available here because the submission-specific audit endpoint `v1/projects/1/forms/<form>/submissions/<uuid>/audits` doesn't currently expand `acteeId` to `actee` like the general audit endpoint `v1/audits` does.